core/chains/evm/txmgr/evm_tx_store_test.go

```diff
@@ -1240,6 +1241,61 @@ func TestORM_UpdateEthTxUnstartedToInProgress(t *testing.T) {
 		err = txStore.UpdateEthTxUnstartedToInProgress(&etx, &attempt)
 		require.ErrorContains(t, err, "eth_tx removed")
 	})
+
+	db = pgtest.NewSqlxDB(t)
+	cfg = newTestChainScopedConfig(t)
+	txStore = cltest.NewTxStore(t, db, cfg)
+	ethKeyStore = cltest.NewKeyStore(t, db, cfg).Eth()
+	_, fromAddress = cltest.MustInsertRandomKeyReturningState(t, ethKeyStore, 0)
+	q = pg.NewQ(db, logger.TestLogger(t), cfg)
+
+	t.Run("update replaces abandoned tx with same hash", func(t *testing.T) {
+		etx := cltest.MustInsertInProgressEthTxWithAttempt(t, txStore, nonce, fromAddress)
+		require.Len(t, etx.EthTxAttempts, 1)
+
+		zero := models.MustNewDuration(time.Duration(0))
+		evmCfg := configtest.NewGeneralConfig(t, func(c *chainlink.Config, s *chainlink.Secrets) {
+			id := utils.NewBig(&cltest.FixtureChainID)
+			ccfg := v2.Defaults(id, &v2.Chain{
+				Transactions: v2.Transactions{
+					ReaperInterval:       zero,
+					ReaperThreshold:      zero,
+					ResendAfterThreshold: zero,
+				},
+			})
+			c.EVM[0] = &v2.EVMConfig{
+				ChainID: id,
+				Enabled: ptr(true),
+				Chain:   ccfg,
+			}
```

Comment:
Do you need to recreate all these settings? Or would it be sufficient to zero out the relevant fields?
```suggestion
			c.EVM[0].ReaperInterval  = zero
			...
```